modules/ztp-configuring-spoke-at-remote-site.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="configuring-edge-cluster-at-remote-location_{context}"]
+= Configuring the edge cluster at the remote location
+
+Perform the following steps at the remote location.
+
+.Prerequisites
+
+* Follow guidance in the supplied quick start guide
+* `kubeadmin` password as supplied by vendor
+* URL of the custom user interface
+* Set name resolution to allow access to the custom user interface
+
+.Procedure
+
+. Unbox the hardware and connect a laptop.
+
+. SSH into control plane node (spoke0) and run the following script.
++
+[source,terminal]
+----
+$
+----
++
+.Example output
++
+[source,terminal]
+----
+
+----
+
+. Open a browser and log in to the edge cluster configuration user interface at the following URL with the supplied `kubeadmin` username and password:
++
+[source,terminal]
+----
+$ https:://edge-cluster-setup.apps.spoke0-cluster.alklabs.com
+----
++
+[NOTE]
+====
+This `kubeadmin` username and password was created at the factory and should have been supplied to you. There is only one user created intitially. 
+====
+
+. Click **Continue**.
+
+. The screens that follow guide you through the intitial setup. 
+
+.. In the first two screens create a new user account by entering a `username` and `password` when prompted. 
++
+[NOTE]
+====
+This new user account is granted ``cluster-admin`` privileges and should be used instead of the factory created account.  
```

Comment:
nit: 
```suggestion
This new user account is granted ``cluster-admin`` privileges and should be used instead of the factory created ``kubeadmin`` account.  
```